src/workloads/scale/LLTQuery.yml

```diff
@@ -0,0 +1,421 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  Workload to Benchmark the effect of LongLivedTransactions on an Update workload.
+
+GlobalDefaults:
+  # These values should match those are the top of LLTPhases.yml
+  dbname: &dbname llt
+  MaxPhases: &MaxPhases 20
+
+  # The Sample Document Shape.
+  Document: &Doc
+    ts: {^Now: {}}
+    caid: {^RandomInt: {min: 0, max: 1000}}
+    cuid: {^RandomInt: {min: 0, max: 100000}}
+    prod: {^RandomInt: {min: 0, max: 10000}}
+    price: {^RandomDouble: {min: 0.0, max: 1000.0}}
+    data: {^Join: {array: ["aaaaaaaaaa", {^FastRandomString: {length: {^RandomInt: {min: 0, max: 10}}}}]}}
+
+  LLTIndexes: &LLTIndexes
+    - keys: {price: 1, ts: 1, cuid: 1}     # PDC
+    - keys: {price: 1, cuid: 1}            # Market
+    - keys: {caid: 1, price: 1, cuid: 1}   # Register
+
+  # Loader Config.
+  LoadThreads: &LoadThreads 4
+  LoadBatchSize: &LoadBatchSize 1000
+
+  # In-memory: Database size works out about 12GB.
+  InitialDocumentCount: &InitialNumDocs 100
+  SecondaryDocumentCount: &SecondaryNumDocs 100
+
+  # WIP make a copy of the workload once these values are tuned correctly and
+  # the workload runs consistently.
+  # Not In-memory: Database size works out about 30GB.
+  # InitialDocumentCount: &InitialNumDocs 49000000
+  # SecondaryDocumentCount: &SecondaryNumDocs 1000000
+  CollectionCount: &CollectionCount 4
+
+  PdcQueryOperation: &PdcQueryOperation
```

Comment:
What does Pdc stand for?

Reply:
Could you add a comment of what these different queries are intending to do and how they stress the system differently?